contributing_to_docs/doc_guidelines.adoc

```diff
@@ -1264,7 +1264,7 @@ EOF
 ----
 Name:               node1.example.com
 Roles:              worker
-Labels:             beta.kubernetes.io/arch=amd64
+Labels:             kubernetes.io/region=amd64
```

Comment:
should not this be `kubernetes.io/arch=amd64` ?

https://kubernetes.io/docs/reference/labels-annotations-taints/#beta-kubernetes-io-arch-deprecated -> source